installer/installer.py

```diff
@@ -994,6 +1068,29 @@ def ask_for_cosmovisor(self):
             self.is_cosmo_needed = False
         else:
             failure_exit(f"Invalid input provided during installation.")
+    
+    def ask_for_cosmovisor_bump(self):
+        self.exec("echo 'export DAEMON_NAME=cheqd-noded' >> ~/.bashrc")
+        self.exec(f"echo 'export DAEMON_HOME={self.home_dir}/.cheqdnode' >> ~/.bashrc")
+        self.exec(f"echo 'export DAEMON_DATA_BACKUP_DIR={self.home_dir}/.cheqdnode' >> ~/.bashrc")
+        std_out = """{}""".format(self.exec("cosmovisor version 2> /dev/null || echo false").stdout)     
+        arr = std_out.split()
+        current_version = None
```

Comment:
1. You can send the output to a file
2. You can read the file, and just read the first line
3. Use regex pattern matching to extract the last word https://www.tutorialspoint.com/How-to-match-at-the-end-of-string-in-python-using-Regular-Expression
4. If present, ditch the v tag (regex already does this) `(\d+)(\.\d+)?(\.\d+)?$`